spec/index.bs

```diff
@@ -7625,6 +7657,12 @@ dictionary GPUSwapChainDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
+Issue: A {{GPUPredefinedColorSpace}} is necessary here.
+We may want this to default to "extended-srgb" if possible.
+It would be the same for non-float-format swapchains, but for float-format swapchains,
+"extended-srgb" is probably the most reasonable behavior:
+float + clamped-srgb just gets you high-bit-depth, but not high-dynamic-range.
```

Comment:
It's too early to speculate about HDR here, and we're not really under pressure to address it in WebGPU yet. Let's stick to SDR for now while we discuss HDR more generally elsewhere.

Reply:
There is no API for HDR output yet - we haven't added float formats yet, so nothing can be done about this issue until we add them. But this is something we want to figure out at the moment we add them, so IMO why not track it here? (If we want to clean up open issues in the spec before 1.0, we have a lot more than just this one.)

Reply:
Unfortunately I don't think isn't something that this group has the background to tackle. I think this should wait on the color folks to hash out what we want, and then apply those findings to webgpu.